proxy/src/http/org.rs

```diff
@@ -0,0 +1,256 @@
+//! Endpoints for Org.
+
+use serde::ser::SerializeStruct as _;
+use serde::{Deserialize, Serialize, Serializer};
+use std::sync::Arc;
+use tokio::sync::RwLock;
+use warp::document::{self, ToDocumentedType};
+use warp::{path, Filter, Rejection, Reply};
+
+use crate::avatar;
+use crate::notification;
+use crate::registry;
+
+/// The org with identifier and avatar
+pub struct Org {
```

Comment:
This should live in `registry.rs`, or alternatively in a seperate `org.rs`. Similar to how we have our own identity, this seems to be upstreams understanding of an org.

Reply:
I moved it to `registry.rs`